emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/PermissionsServiceImpl.java

```diff
@@ -226,21 +238,14 @@ private boolean chmodCollection(DataGridPermType permType, String path, boolean
 		try {
 			logger.debug("Setting {} permission on collection {} for user/group as ADMIN{}", permType, path, uName);
 
+			// TODO expand this
```

Comment:
Do we need to make an issue to link here?

Reply:
Yes please. Or remove it.

Example:
```java
// TODO(#<issue_number>): Words ...
```

Reply:
I believe this has been handled.  I think that was just a note to myself that should now be removed.  I removed it.